libs/email/project.json

```diff
@@ -5,5 +5,7 @@
   "projectType": "library",
   "tags": [],
   "// targets": "to see all targets run: nx show project email --web",
-  "targets": {}
+  "targets": {
+    "dev": { "command": "email dev --dir libs/email/src/components" }
```

Comment:
**question:** What's the purpose of this? I am not experienced with Resend, so I need some explanation. :)

Reply:
it's not important right now, but it's how we can preview the email template that we already have.

you can learn more about it in the [CLI docs](https://react.email/docs/cli)

Reply:
Great, thanks! :)